spec/services/vot/parser_spec.rb

```diff
@@ -52,51 +74,70 @@
       end
     end
 
-    it 'adds the two pieces of fair evidence components' do
-      vector_of_trust = 'Pb'
+    context 'when two pieces of fair evidence is required' do
```

Comment:
"are required"?

Reply:
"two pieces of fair evidence" is a single requirement, so i used the singular